base/base/defines.h

```diff
@@ -31,6 +31,8 @@
 #   endif
 #endif
 
+// more aliases: https://mailman.videolan.org/pipermail/x264-devel/2014-May/010660.html
+
 #if defined(_MSC_VER)
```

Comment:
BTW, we don't even pretend to work on Windows, we can remove this macro.

Reply:
Completely agree. Will fix it shortly.